zetaclient/chains/evm/observer/inbound.go

```diff
@@ -34,6 +35,76 @@ import (
 	"github.com/zeta-chain/node/zetaclient/zetacore"
 )
 
+var (
+	// erc20AddressToForeignCoinAssetMap maps the chain id and foreign ERC20 address to the coin asset string
```

Comment:
```suggestion
	// erc20AddressToForeignCoinAssetMap maps the chain id and foreign ERC20 address to the coin asset string
	// this is currently necessary because of the following issue: https://github.com/zeta-chain/node/issues/3274
```